### PR TITLE
Add force mobile UI

### PR DESCRIPTION
**Pull request by Justicea83**
Not every use case requires the fullscreen, so i added these changes so we can force mobile Ui

### CODE REVIEW

File: src/factory.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { PlaidLinkOptions, Plaid } from './types';
+import {PlaidLinkOptions, Plaid, PlaidAdditionalConfig} from './types';
```
Comment (marcaaron) [inline src/factory.ts:1]:
```suggestion
import { PlaidLinkOptions, Plaid, PlaidAdditionalConfig } from './types';
```

File: src/types/index.ts [hunk 2]
```diff
@@ -137,6 +137,11 @@ export type PlaidLinkOptionsWithPublicKey = CommonPlaidLinkOptions & {
   paymentToken?: string;
 };
 
+export type PlaidAdditionalConfig = {
+  // Specify if we should for the view like on a mobile device
+  forceMobile?: boolean;
+}
```
Comment (marcaaron) [inline src/types/index.ts:143]:
```suggestion
};
```

File: src/usePlaidLink.ts [hunk 3]
```diff
@@ -2,7 +2,7 @@ import { useEffect, useState } from 'react';
 import useScript from 'react-script-hook';
 
 import { createPlaid, PlaidFactory } from './factory';
-import { PlaidLinkOptions, PlaidLinkOptionsWithPublicKey } from './types';
+import {PlaidAdditionalConfig, PlaidLinkOptions, PlaidLinkOptionsWithPublicKey} from './types';
```
Comment (marcaaron) [inline src/usePlaidLink.ts:5]:
```suggestion
import { PlaidAdditionalConfig, PlaidLinkOptions, PlaidLinkOptionsWithPublicKey } from './types';
```

**Review by marcaaron — COMMENTED** [review]
Saw some style issues. Is there a linked issue we can attach to this?

**Comment by perfectsimulation** [comment]
This project already has the lockfile `yarn.lock`. The `package-lock.json` here should be removed.

File: src/types/index.ts [hunk 4]
```diff
@@ -137,6 +137,11 @@ export type PlaidLinkOptionsWithPublicKey = CommonPlaidLinkOptions & {
   paymentToken?: string;
 };
 
+export type PlaidAdditionalConfig = {
+  // Specify if we should for the view like on a mobile device
```
Comment (zkirby) [inline src/types/index.ts:141]:
```suggestion
  // Specify if we should force the view to be as if it was on a mobile device.
```

**Comment by Justicea83** [comment]
Gentle bump

**Comment by Justicea83** [comment]
Updated, please rereview

**Comment by Justicea83** [comment]
Gentle bump

**Comment by Justicea83** [comment]
Gentle bump 🙏🏻

**Comment by Justicea83** [comment]
Please am bumping again

**Comment by phoenixy1** [comment]
@Justicea83 I'm sorry for the (very unreasonable) delay in responding to this PR, and thank you for submitting it! 

In this particular case, `forceMobile` is not officially supported (we use it for internal testing only) so we won't be accepting this particular PR as we don't want to encourage folks to use unsupported options.

**Comment by FPLCodes** [comment]
It's been over 3 years now do you guys have an official way to do this yet?

**Pull request closed without merging**